src/SFA.DAS.EmployerFinance.Web.UnitTests/Orchestrators/WhenGettingTransfersFinancialBreakdown.cs

```diff
@@ -0,0 +1,81 @@
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerFinance.Web.UnitTests.Orchestrators
+{
+    [TestFixture]
+    public class WhenGettingTransfersFinancialBreakdown
+    {
+        private TransfersOrchestrator _orchestrator;
+        private Mock<IAuthorizationService> _authorisationService;
+        private Mock<IHashingService> _hashingService;
+        private Mock<IManageApprenticeshipsService> _maService;
+        private Mock<IAccountApiClient> _accountApiClient;
+        private Mock<IFeatureTogglesService<EmployerFeatureToggle>> _featureTogglesService;
+
+        private const string HashedAccountId = "123ABC";
+        private const long AccountId = 1234;
+
+        [SetUp]
+        public void Setup()
+        {
+            _authorisationService = new Mock<IAuthorizationService>();
+            _hashingService = new Mock<IHashingService>();
+            _maService = new Mock<IManageApprenticeshipsService>();
+            _accountApiClient = new Mock<IAccountApiClient>();
+            _featureTogglesService = new Mock<IFeatureTogglesService<EmployerFeatureToggle>>();
+
+            _hashingService.Setup(h => h.DecodeValue(HashedAccountId)).Returns(AccountId);
+            _featureTogglesService.Setup(x => x.GetFeatureToggle(It.IsAny<string>())).Returns(new EmployerFeatureToggle { IsEnabled = true });
+
+            _orchestrator = new TransfersOrchestrator(_authorisationService.Object, _hashingService.Object, _maService.Object, _accountApiClient.Object, _featureTogglesService.Object);
+        }
+
+        [Test]
+        public async Task CheckFinancialBreakdownViewModel()
+        {
+            var financialBreakdownResponse = new GetFinancialBreakdownResponse
+            {
+                AcceptedPledgeApplications = 20000,
+                ApprovedPledgeApplications = 10000,
+                Commitments = 1000,
+                TransferConnections = 1000,
+                NumberOfMonths = 12
+            };
+
+            _maService.Setup(o => o.GetFinancialBreakdown(AccountId)).ReturnsAsync(financialBreakdownResponse);
+
+            SetupTheAccountApiClient();
+
+            var actual = await _orchestrator.GetFinancialBreakdownViewModel(HashedAccountId);
+
+            Assert.AreEqual(financialBreakdownResponse.AcceptedPledgeApplications, actual.Data.AcceptedPledgeApplications);
+            Assert.AreEqual(financialBreakdownResponse.ApprovedPledgeApplications, actual.Data.ApprovedPledgeApplications);
+            Assert.AreEqual(financialBreakdownResponse.Commitments, actual.Data.Commitments);
+            Assert.AreEqual(financialBreakdownResponse.TransferConnections, actual.Data.TransferConnections);
+            Assert.AreEqual(financialBreakdownResponse.NumberOfMonths, actual.Data.NumberOfMonths);
+        }
+
+        private void SetupTheAccountApiClient()
+        {
+            var modelToReturn = new AccountDetailViewModel
+            {
+                ApprenticeshipEmployerType = "Levy"
```

Comment:
It's not clear to me what difference this makes?

Reply:
yes true, I have removed that method and refactored the code to set up the _accountApiClient.